### PR TITLE
Spell check

### PR DESCRIPTION
**Pull request by jsoref**
This adds [check-spelling](https://check-spelling.dev/)`@v0.0.20-alpha3` using [check-spelling/spell-check-this@prerelease](https://github.com/check-spelling/spell-check-this/tree/prerelease/.github).

I'm not including experimental update. We're using it internally, but I think it's still a bit too rough.

You'll want to review the expect file as it's essentially the current baseline. Note this PR includes one extra typo fix for an item that snuck in between when I started #365 and when it was merged.

Any open PRs should be poked (force-push rebase is probably the best approach, although adding a null commit would also work) or else you risk them adding typos and confusing people.

I'm generally quite available if you need help (I respond to `@`, and I'm reachable on irc, or by email).

### CODE REVIEW

**Review by cmouse — APPROVED** [review]

File: .github/actions/spelling/expect.txt [hunk 1]
```diff
@@ -0,0 +1,1538 @@
...
+bayes
+baz
+BBAAAAAAAAAAAAAAAAAAAAAAB
+bcc
+bcn
+bcrypt
+BDAT
+bdb
+bernat
+Besidesm
```
Comment (cmouse) [inline .github/actions/spelling/expect.txt:112]: fixable typo

Comment (cmouse) [reply .github/actions/spelling/expect.txt:112]: `Besidesm if the system was just` => `Besides if the system was just`

File: .github/actions/spelling/expect.txt [hunk 2]
```diff
@@ -0,0 +1,1538 @@
...
+blitiri
+blkid
+blockdev
+blog
+bodystructure
+Boehm
+Bokmal
+bootup
+BROKENCHAR
+bruteforce
```
Comment (cmouse) [inline .github/actions/spelling/expect.txt:130]: `bruteforce` => `brute force`

File: .github/actions/spelling/expect.txt [hunk 3]
```diff
@@ -0,0 +1,1538 @@
...
+userdb
+userdn
+userguide
+userid
+userinfo
+userip
+userlist
+username
+userslist
+Uset
```
Comment (cmouse) [inline .github/actions/spelling/expect.txt:1432]: This is also typo, it should be User.

File: .github/actions/spelling/expect.txt [hunk 4]
```diff
@@ -0,0 +1,1538 @@
...
+usr
+utf
+utils
+UUID
+uuuuuu
+Uvh
+uzing
```
Comment (cmouse) [inline .github/actions/spelling/expect.txt:1439]: This is a typo, it should be `using`

File: .github/actions/spelling/expect.txt [hunk 5]
```diff
@@ -0,0 +1,1538 @@
...
+versionchanged
+Versioning
+versionremoved
+vfile
+vfuncs
+vgdb
+vhost
+virtualdate
+virtualenv
+virtualisation
```
Comment (cmouse) [inline .github/actions/spelling/expect.txt:1460]: not sure if this should be virtualization.

**Review by cmouse — CHANGES_REQUESTED** [review]

**Comment by cmouse** [comment]
Please remove the open discussions from expect.txt, and close them.

File: .github/actions/spelling/expect.txt [hunk 6]
```diff
@@ -0,0 +1,1538 @@
...
+lowwater
+lport
+lrs
+lrw
+lrwstipeka
+lrwstipekxa
+lrwstipekxacd
+lsearch
+lsub
+LTMP
```
Comment (cmouse) [inline .github/actions/spelling/expect.txt:743]: this should be LMTP

File: .github/actions/spelling/expect.txt [hunk 7]
```diff
@@ -0,0 +1,1538 @@
...
+nouser
+NOVALGRIND
+nproc
+nscd
+nsend
+nsname
+nss
+Ntll
+ntlm
+ntml
```
Comment (cmouse) [inline .github/actions/spelling/expect.txt:918]: should be ntlm

**Review by sirainen — CHANGES_REQUESTED** [review]

File: .github/actions/spelling/expect.txt [hunk 8]
```diff
@@ -0,0 +1,1538 @@
...
+altstorage
+Amavis
+amavisd
+amazonaws
+amz
+amzn
+AND'ing
+anotherhost
+Antiabuse
+antispam
```
[the remark this answers is not in the document]
Comment (jsoref) [reply .github/actions/spelling/expect.txt:41]: The former redirects from:
https://oxpedia.org/wiki/index.php?title=AppSuite:Dovecot_Antiabuse_Shield =>
https://oxpedia.org/wiki/index.php?title=AppSuite:OX_Abuse_Shield

[the remark this answers is not in the document]
Comment (jsoref) [reply .github/actions/spelling/expect.txt:41]: The latter is doable, but it's a non-trivial change (involving renaming a file):
```
source/configuration_manual/howto/antispam_with_sieve.rst:.. _howto-antispam_with_imapsieve:
source/configuration_manual/howto/antispam_with_sieve.rst:Replacing antispam plugin with IMAPSieve
source/configuration_manual/howto/antispam_with_sieve.rst:https:/ /
source/configuration_manual/sieve/plugins/imapsieve.rst:See :ref:`Replacing antispam plugin with
source/configuration_manual/sieve/plugins/imapsieve.rst:IMAPSieve <howto-antispam_with_imapsieve>`
source/configuration_manual/spam_reporting.rst:.. seealso:: :ref:`howto-antispam_with_imapsieve`
```

You can also make changes such as this after this merges.

(I'll do the former now.)

**Review by cmouse — DISMISSED** [review]

**Comment by jsoref** [comment]
Checks for the latest merge failed: https://github.com/jsoref/dovecot-documentation/commit/6b7ddeb9b5badf2da58f2266ee151f2dcbfa1789#commitcomment-62676671

**Comment by cmouse** [comment]
Looks like it still complains about ECDHE.?

**Review by sirainen — APPROVED** [review]

**Pull request merged**